backend/functions/offices/offices.function.test.ts

```diff
@@ -1,27 +1,31 @@
-import httpTrigger from '../offices/offices.function';
 import { CamsError } from '../lib/common-errors/cams-error';
-import { createMockAzureFunctionRequest } from '../azure/functions';
-import ContextCreator from '../lib/adapters/utils/application-context-creator';
+import ContextCreator from '../azure/application-context-creator';
```

Comment:
We should not be reaching into `/backend/functions/azure` from anywhere in `/lib`.